samples/azure-quantum/variational-quantum-eigensolver/VQE-qiskit-hydrogen-quantinuum-emulator.ipynb

```diff
@@ -0,0 +1,717 @@
+   "source": [
+    "## 2. Run on Quantinuum emulator via Azure Quantum Workspace\n",
+    "\n",
+    "Now, you can connect to the Azure Quantum Workspace and run VQE on the hardware backends."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "id": "c042c41e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Connect to the Azure Quantum Workspace via a Qiskit provider\n",
+    "from azure.quantum.qiskit import AzureQuantumProvider\n",
+    "provider = AzureQuantumProvider(\n",
+    "    resource_id = \"\",\n",
+    "    location = \"\"\n",
+    ")\n",
+    "\n",
+    "# Create Quantinuum emulator backend\n",
+    "quantinuum_emulator = provider.get_backend(\"quantinuum.hqs-lt-s1-sim\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "69a3a14c",
+   "metadata": {},
+   "source": [
+    "### Estimate cost\n",
+    "\n",
+    "You can now estimate how many credits (\"HQC\"/\"EHQC\" for Quantinuum) it will cost to run VQE. For more information about pricing, see the [Azure Quantum pricing](https://docs.microsoft.com/azure/quantum/pricing) documentation page."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 83,
+   "id": "ed975803",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cost = [quantinuum_backend.estimate_cost(circ, shots=1000) for circ in ready_circs]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 84,
+   "id": "9edf5289",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "16.6 EHQC\n",
+      "16.2 EHQC\n"
+     ]
+    }
+   ],
+   "source": [
+    "for _cost in cost:\n",
+    "    print(_cost.estimated_total, _cost.currency_code)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8924df25",
+   "metadata": {},
+   "source": [
+    "So, given approx. 300 energy evaluations, this would give a total cost"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 87,
+   "id": "627268da",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(32.8, 9840.0)"
+      ]
+     },
+     "execution_count": 87,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "energy_eval_cost = sum(_cost.estimated_total for _cost in cost)\n",
+    "energy_eval_cost, energy_eval_cost * 300"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 91,
+   "id": "13e44adb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cost = [ionq_qpu_backend.estimate_cost(circ, shots=1000) for circ in ready_circs]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 92,
+   "id": "451f1310",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1.44 USD\n",
+      "1.38 USD\n"
+     ]
+    }
+   ],
+   "source": [
+    "for _cost in cost:\n",
+    "    print(_cost.estimated_total, _cost.currency_code)"
```

Comment:
Same comment as above, I think this cell is currently printing IonQ costs, but we can probably remove these two cells from this sample!